lib/pages/splash/splash_page.dart

```diff
@@ -3,25 +3,38 @@ import 'package:flutter/scheduler.dart';
 import 'package:flutter/widgets.dart';
 import 'package:get/get.dart';
 import 'package:survey/navigator/navigator.dart';
+import 'package:survey/pages/splash/splash_controller.dart';
 
 class SplashPage extends StatelessWidget {
   final AppNavigator _appNavigator = Get.find();
+  final splashController = Get.put(SplashController());
 
   @override
   Widget build(BuildContext context) {
     if (SchedulerBinding.instance.schedulerPhase == SchedulerPhase.idle) {
       SchedulerBinding.instance
-          .addPostFrameCallback((_) => _appNavigator.popAndNavigateToLogin());
+          .addPostFrameCallback((_) => _loginStatusCheck());
```

Comment:
good to know 👍

Reply:
I think normally people choose to implement the Stateful widget to get the callback after init, not sure if this way has any drawback tho. 😄

Reply:
I did some research and this is one of recommend ways to handle listener without using Stateful widget.
For other state management libs, we have `ProviderListener` (Provider, Riverpod) or `BlocListener` (Bloc) widget to listen the changes but so sad I cannot find any similar things in GetX 😅